webpack.config.client.js

```diff
@@ -11,7 +11,7 @@ const TerserPlugin = require('terser-webpack-plugin');
 const LoadablePlugin = require('@loadable/webpack-plugin');
 const webpack = require('webpack');
 const dotenv = require('dotenv');
-const { DuplicatesPlugin } = require('inspectpack/plugin');
+const DuplicatePackageCheckerPlugin = require('duplicate-package-checker-webpack-plugin');
```

Comment:
This plugin was not listing the duplicate entries, so I've now replaced it with this - it's on the WebPack webiste as a recommended plugin; https://webpack.js.org/awesome-webpack/#webpack-plugins (search for Duplicate)